### ISSUE TITLE
one column per language

### ISSUE THREAD

**Issue by gabriben** [aghasemi/swissdemonyms#2]
should there be one column per language, since a lot of cities have dual/triple demonyms in different languages?

**Comment by aghasemi** [comment]
Yes, it makes sense.  First I need the list of official languages per Gemeinde/Commune. It is in the backlog, but let's for now just keep all in one column
